.travis.yml

```diff
@@ -35,7 +35,6 @@ env:
     - COVERITY_EMAIL="sjudeng@gmail.com"
     # Default Elasticsearch heap size can be too large for Travis
     - ES_JAVA_OPTS="-Xms256m -Xmx512m"
-    - export STRUCTOR_LATEST_TAG=v0.4.0
```

Comment:
What does this variable did? It's no longer necessary?

Reply:
This variable is now configured in the travis ci web frontend.

Reply:
I will do CTR commit tomorrow to remove it from all release branches.

Reply:
https://github.com/JanusGraph/janusgraph/commit/8c9db3d1e7f421179468a73eca294fa416d99907